src/isIsogram.test.js

```diff
@@ -8,12 +8,27 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('')).toBe('boolean');
+  });
+
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('PIG')).toBe(true);
+  });
```

Comment:
You don't check case sensitivity here, you should use an input which has the same letter in different cases, like `'Aa'`